core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceRequestConverterTest.java

```diff
@@ -100,14 +101,19 @@ public String convert2(RequestJsonObj1 obj1) {
         }
 
         @Post("/convert3")
-        public String convert3(@RequestConverter(TestRequestConverterOptional1.class)
-                                       Optional<RequestJsonObj1> obj1,
-                               @RequestConverter(TestRequestConverterOptional2.class)
-                                       Optional<RequestJsonObj2> obj2) {
+        public String convert3(@RequestConverter(TestRequestConverter1.class) Optional<RequestJsonObj1> obj1,
+                               @RequestConverter(TestRequestConverter2.class) Optional<RequestJsonObj2> obj2) {
             assertThat(obj1.isPresent()).isTrue();
             assertThat(obj2.isPresent()).isTrue();
             return obj2.get().strVal();
         }
+
+        @Post("/convert4")
+        @RequestConverter(NullReturningConverter.class)
+        public void convert4(Optional<String> optional, @Nullable String nullable) {
```

Comment:
Question: What is the expected behavior if `String nullable` is not annotated by `@Nullable`?
`null` value is injected into `nullable`.

Reply:
By the way, `NullReturningConverter` returns null. It looks like a correct test.

Reply:
I added another method with non-null parameter which is expected to fail with a 400 Bad Request error.